tests/integration/cli/classification/test_classification.py

```diff
@@ -105,6 +105,14 @@ def test_otx_train(self, template, tmp_dir_path):
         args1["--load-weights"] = f"{template_work_dir}/trained_{template.model_template_id}/weights.pth"
         otx_train_testing(template, tmp_dir_path, otx_dir, args1)
 
+    @e2e_pytest_component
```

Comment:
Is it okay, we set gpus like this? If ci machine run multiple tests, then the ci machine could give a lot of burden for specific gpu

Reply:
you're right. but current CI doesn't run multiple job at the same time.